.github/actions/fedora-build-jemalloc/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Build jemalloc (Fedora only)'
+description: 'Build jemalloc on Fedora with enable-jemalloc-prefix'
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        dnf -y install autoconf make gcc awk curl unzip
+        curl -L -o jemalloc-5.3.0.zip https://github.com/jemalloc/jemalloc/archive/refs/tags/5.3.0.zip
+        unzip jemalloc-5.3.0.zip
+        cd jemalloc-5.3.0
+        ./autogen.sh
+        ./configure --prefix=/opt/jemalloc --enable-jemalloc-prefix
+        make -j$(nproc)
+        make install
+      shell: bash
```

Comment:
We include a vendored jemalloc so we don't need to download it. Can we not just alter the make file to pass these options during the building of our vendored one?

Reply:
🥲 Sorry for using a more complicated solution earlier. I’ve found that it’s actually sufficient to just install awk properly on Fedora.